index.html

```diff
@@ -11872,9 +11872,14 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
 	  <h2>Including Elements in the Accessibility Tree</h2>
           <p>If not already excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Text nodes</li>
-            <li>Elements that may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>.</li>
-            <li>Elements that are focusable, or have an ID <a>attribute</a> and an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the implicit or explicit <a>semantics</a> of the required context role. In either case, the element may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+            <li>Text nodes.</li>
+            <li>Elements that may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>.
+              <ul>
+                <li>Elements that are focusable may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+                <li>Elements that have an ID <a>attribute</a>, and are owned by or have an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the ID, may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+                <li>Elements, including their descendants, that receive focus and fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event even though they have <sref>aria-hidden</sref> set to <code>true</code>.</li>
```

Comment:
This doesn't seem right to me. In my reading this states that if there is an element which can receive focus and fire an accessibility API focus event then all of its descendents MUST be exposed.

Reply:
I think that is a correct reading. It is probably necessary to ALWAYS include descendants of focusable elements. I imagine a lot of marginally accessible things would break if that were not the case.

Reply:
> Elements, including their descendants, that receive focus and fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event even though they have <sref>aria-hidden</sref> set to <code>true</code>.

Editorial: I might phrase this:

Any _rendered_ element that becomes focused, even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.

Reply:
The "rendered" bit may be important, because it's ambiguous what it means to focus an element that is not in the render tree. Normally this clarification would not be necessary, but juxtaposed with this reference to `aria-hidden` the spec should be clear.

Reply:
Changed to use your words, with the "fire an accessibility API focus event" bit back in.